app.js

```diff
@@ -205,7 +204,6 @@ app.get('/api/v1/geo', cors(), resources.v1.geo.get)
 
 app.get('/services/geoip', resources.services.geoip.get)
 
-app.options('/services/images', cors())
```

Comment:
I don't think the OPTIONS "pre-flight" request is enabled for `/services` since we are enabling all preflight requests with the path `/api/*` so we would probably still need this line of code.

Reply:
You're right -- good catch! Thanks.